### PR TITLE
New package: zscroll-2.0.1

### PR DESCRIPTION
**Pull request by Barbaross93**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (amd64)

### CODE REVIEW

**Comment by Barbaross93** [comment]
The build errors are the result of `python3` not being found. I'm not sure if there's something I need to change on my end?

File: srcpkgs/zscroll/template [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Template file for 'zscroll'
+pkgname=zscroll
+version=2.0.1
+revision=1
+build_style=python3-module
```
Comment (icp1994) [inline srcpkgs/zscroll/template:5]:
```suggestion
build_style=python3-module
make_check_args=" -s"
hostmakedepends="python3-setuptools"
depends="python3"
checkdepends="python3-pytest"
```

File: srcpkgs/zscroll/template [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+short_desc="Text scroller for panels or terminals"
+maintainer="Barbaross <cullen.ross@proton.me>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/noctuid/zscroll"
+distfiles="${homepage}/archive/refs/tags/${version}.tar.gz"
```
Comment (icp1994) [inline srcpkgs/zscroll/template:10]:
```suggestion
distfiles="https://github.com/noctuid/zscroll/archive/refs/tags/${version}.tar.gz"
```

**Review by icp1994 — CHANGES_REQUESTED** [review]

File: srcpkgs/zscroll/template (hunk 2 above)
Comment (Barbaross93) [reply srcpkgs/zscroll/template]: Thank you for your feedback!

**Pull request closed without merging**